### PR TITLE
Faqs-Q2

### PR DESCRIPTION
**Pull request by kartikeya2609**
Faqs-Q2

Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: Added few FAQs

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.

### CODE REVIEW

**Comment by dewan-ahmed** [comment]
Hi @kartikeya2609 please check the build fail details and fix the issues. Then we can review. Thanks.

File: kb/security-testing-orchestration/security-testing-orchestration-faqs.md [hunk 1]
```diff
@@ -46,6 +46,25 @@ For more details, go to [What's supported](/docs/security-testing-orchestration/
...
 For STO to successfully ingest your scan results, the ingestion file must adhere to the specific JSON format described here: [Ingest from unsupported scanners](/docs/security-testing-orchestration/custom-scanning/ingesting-issues-from-other-scanners)
 
+## Do user need root access to run Snyk scans in STO?
+If you need to add trusted certificates to your scan images at runtime, you must run the scan step with root access. However, you can configure your STO scan images and pipelines to run scans as non-root and establish trust for your proxies using custom certificates.
+
+## What is the default workspace path for the Scan?
+The default workspace path is /harness. You can override this if you want to scan a subset of the workspace.
+
+## How can user configure authentication for scans?
+Access Token (Orchestration scans): Use a Harness text secret for your encrypted token and reference it in the format <+secrets.getValue("myaccesstoken")>.
```
Comment (krishi0408) [inline kb/security-testing-orchestration/security-testing-orchestration-faqs.md:56]:
```suggestion
Access Token (Orchestration scans): Use a Harness text secret for your encrypted token and reference it in the format `<+secrets.getValue("myaccesstoken")>`.
```

File: kb/continuous-delivery/faqs.md [hunk 2]
```diff
@@ -3236,6 +3236,24 @@ AIDA is enabled on the Harness platform by default. To read more on the security
...
+#### If user re-run the execution from failed stage and the failed stage is configured with the chained pipeline, so the whole child pipeline will re-run or only the specific step inside the child pieline will re-run?
+So if you are re-running the pipeline from the failed stage. So the whole child pipeline will considered as the stage and it will re-run the entire child pipeline.
+
+#### Harness uses which encoding in the copy command step?
+Harness uses UTF-8 encoding in the copy config zommand step.
+
+#### How long are deployment instances tracked in dashboard?
+On the dashboard is limited to active instances from the past 30 days.
+
+#### Can user use the \*.tgz file to do the helm template cli?
```
Comment (krishi0408) [inline kb/continuous-delivery/faqs.md:3251]:
```suggestion
#### Can user use the `\*.tgz` file to do the helm template cli?
```

File: kb/continuous-delivery/faqs.md [hunk 3]
```diff
@@ -3236,6 +3236,24 @@ AIDA is enabled on the Harness platform by default. To read more on the security
 Yes, we can use exported variables to reference a step group variable without knowing the step group ID.
 One can follow the syntax : `<+exportedVariables.getValue("stepGroup.ALIAS_NAME.OUTPUT_VARIABLE_NAME")>` This method allows you to reference the variable by its alias name instead of needing to know the step group ID. For more details, you can refer to the Harness documentation on [Scoping output variables using aliases](https://developer.harness.io/docs/continuous-delivery/x-platform-cd-features/cd-steps/utilities/shell-script-step/#scoping-output-variables-using-aliases)
 
+#### Can user provide the user group in the Approval step as expression?
+Yes, user can give the expression at the place of user group filed in Approval step.
+
+#### If user re-run the execution from failed stage and the failed stage is configured with the chained pipeline, so the whole child pipeline will re-run or only the specific step inside the child pieline will re-run?
+So if you are re-running the pipeline from the failed stage. So the whole child pipeline will considered as the stage and it will re-run the entire child pipeline.
+
+#### Harness uses which encoding in the copy command step?
```
Comment (krishi0408) [inline kb/continuous-delivery/faqs.md:3245]:
```suggestion
#### Which encoding in the copy command step in Harness?
```

**Review by krishi0408 — APPROVED** [review]

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 4]
```diff
@@ -2349,6 +2349,18 @@ curl -X POST -i -H "Authorization: Bearer AUTH_TOKEN" https://<DRONE_SERVER_FQDN
...
+### How can user Spin up a dynamic containers during testing after the build?
+User can use the Background step as suggested in this doc: https://developer.harness.io/docs/continuous-integration/use-ci/manage-dependencies/background-step-settings/
+
+### If user have test results as part of a generic Run step with mvn clean deploy, would user be able to slurp those test results over into the test Dashboard still? Or is the Unit Test step the only way to get them on that tab?
+Yes, you can use the Run step to run your maven command, and upload the test results. There is a field for this in the run step (like in the run tests step). Doc for refrence:
+https://developer.harness.io/docs/continuous-delivery/x-platform-cd-features/cd-steps/containerized-steps/run-step/#report-paths 
+The RunTests allow you to not only run your tests, but also to use Test Intelligence to run only unit tests that are relevant to the code changes made, thus cutting down the test cycle time.
+
+
+### Is user can filter which CI builds timed out during Initialize step?
```
Comment (nofarblue) [inline kb/continuous-integration/continuous-integration-faqs.md:2361]: I  think tis is very specific , and shouldn't be included in FAQ

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 5]
```diff
@@ -2349,6 +2349,18 @@ curl -X POST -i -H "Authorization: Bearer AUTH_TOKEN" https://<DRONE_SERVER_FQDN
 ### What actions can I take to address the extended runtime of a Docker command, particularly when encountering timeouts during microdnf updates?
 The prolonged runtime of Docker commands, often exacerbated by timeouts during microdnf updates, can be primarily attributed to network issues. To address this, you should verify the connectivity of the repomirror being utilized. Additionally, you could examine and potentially adjust the timeouts configured on the repomirror to enhance performance. These steps can help mitigate the impact of network-related delays and improve the efficiency of Docker operations.
 
+### How can user Spin up a dynamic containers during testing after the build?
```
Comment (nofarblue) [inline kb/continuous-integration/continuous-integration-faqs.md:2352]: I understand the answer, but I don't think the question is clear. there are many use cases for background step. 
Maybe something like ' how can I spin up a service needed by my build steps?'

**Review by nofarblue — CHANGES_REQUESTED** [review]
I've made multiple comments on the CI FAQ updates. please review.

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!

**Pull request closed without merging**